schemas/CodiusManifestSpec.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
Lets call this just CodiusSpec since we are generating the manifest that will be uploaded.